misago/users/management/commands/removeips.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandError
+
+from misago.core.utils import ANONYMOUS_IP
+from misago.conf.defaults import MISAGO_IP_STORE_TIME
+from misago.users.signals import anonymize_old_ips
+
+
+class Command(BaseCommand):
+    help =  "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."
+    
+    def handle(self, *args, **options):
+      if MISAGO_IP_STORE_TIME == None:
+        raise CommandError('Anonymization of the old user IPs is currently disabled')
```

Comment:
Please use `not` instead of `== None`. Also, this is not an error, just information message that anonymization is disabled, `self.stdout.write()` followed by `return` will do instead. Lastly, Misago convention is to use double quotes for human-readable messages.